akka-discovery/src/main/scala/akka/discovery/ServiceDiscovery.scala

```diff
@@ -320,11 +328,14 @@ abstract class ServiceDiscovery {
   /**
    * Java API
    *
+   * Convenience lookup accepting a name. If the name is a valid srv entry a SRV lookup is done, otherwise
+   * if not a regular lookup. For more control use the overload accepting a [[Lookup]]
```

Comment:
```suggestion
   * Convenience lookup accepting a name. If the name is a valid SRV entry, an SRV lookup is done, otherwise
   * a regular lookup. For more control use the overload accepting a [[Lookup]].
```